turtlebot3c-bringup-snap/snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: turtlebot3c-bringup
-adopt-info: workspace
+version: '0.1'
```

Comment:
Let's make it semver-like `0.1.0`

Reply:
Actually, same question as for [humble](https://github.com/canonical/turtlebot3c-snap/pull/76#discussion_r1840910091)